### DISCUSSION TITLE
Issued Documents/receipts cancellati

### DISCUSSION THREAD

**Discussion by giovannizorzato** [fattureincloud/api#340] [Q&A] [answered]
È possibile capire se/quando una fattura/nota di credito/ricevuta è stata cancellata o semplicemente non verrà più restituita nella lista?
Non è possibile sincronizzare i dati correttamente con altre fonti senza poter accedere ai documenti cancellati.
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @giovannizorzato, a breve verranno aperti i webhooks sugli eventi relativi ai documenti emessi, attraverso questi sarà possibile essere notificati dell'eliminazione dei documenti, attualmente l'unica soluzione credo sia quella di tenere traccia ad ogni chiamata di list degli id ritornati e confrontarli con quelli salvati precedentemente.

Grazie,
Emanuele.

**Reply by dalpo** [reply]
@emajo Ti ringrazio per la risposta data, visto che mi sono imbattuto nella stessa problematica.

Oltre ai webhook, sarebbe molto utile che nelle chiamete API venisse esposto un qualche attributo con cui indicare lo stato "cancellato" del documento. 
Ad esempio un `deleted_at`con il timestamp di cancellazione, in maniera analoga ai vari `created_at` o `updated_at`.

Grazie,
Andrea
